numpy/random/tests/test_generator_mt19937.py

```diff
@@ -987,6 +987,15 @@ def test_beta(self):
              [2.830254190078299e-04, 1.744709918330393e-01]])
         assert_array_almost_equal(actual, desired, decimal=15)
 
+    def test_beta_small_a_and_b(self):
```

Comment:
Could you remind me why this test of `beta` is part of this pull request?

Reply:
On my local build I have experienced slowness for the stick breaking code branch when the alpha.max() < 1.0e-10. 

With the new beta test I was able to track this down to slowness in the beta generator.  With a clean install of numpy-1.16.5 this slowness does not occur (for np.random.beta) so either the beta implementation has fundamentally changed (I don't think so) or the build process on my machine (macOS) is doing something that results in this slowness for my local build only (I assume this is the case). 

I would be much obliged if you could double check whether the slowness is gone for your local build. In that case I would drop the comments about slowness in the test. I think the test is a nice to have reference, but if it is out of the scope of this PR I can remove it.

Reply:
OK, it looks like the beta generator does get very slow when its argument are tiny.  That should be raised as a new issue;  I don't think we need to deal with that issue here.  This PR does not affect the beta distribution, so adding a test for it here is unnecessary.